spec/support/ansible_shared/automation_manager/job.rb

```diff
@@ -29,7 +29,7 @@
       'cloud_credential_id'   => cloud_credential.manager_ref,
       'network_credential_id' => network_credential.manager_ref
     ).tap do |rjob|
-      allow(rjob).to receive(:stdout).and_return('job stdout')
+      allow(rjob).to receive(:stdout).with('html').and_return('job stdout')
```

Comment:
agreed...if you are going to stub an html response, the test should return something that resembles html.